docs/serving/autoscaling/supported-autoscaler-types.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Supported Autoscaler types"
+linkTitle: "Supported Autoscaler types"
+weight: 10
+type: "docs"
+---
+
+Knative Serving supports the implementation of Knative Pod Autoscaler (KPA) and Kubernetes' Horizontal Pod Autoscaler (HPA). The features and limitations of each of these Autoscalers are listed below.
+
+## KPA
+* Part of the Knative Serving core and enabled by default.
+* Supports scale to zero functionality.
+* Does not support CPU-based autoscaling.
+
+## HPA
+* Not part of the Knative Serving core, and must be enabled during Knative Serving installation.
+<!-- Question: do we provide details in the install docs about this?-->
```

Comment:
@markusthoemmes do we need to add somethng about HPA to the install docs?

Reply:
AFAIK, there's a section explaining how to install it there already.

Reply:
You could link here https://knative.dev/docs/install/any-kubernetes-cluster/#optional-serving-extensions